src/components/Tooltip/Tooltip.styles.ts

```diff
@@ -0,0 +1,49 @@
+import fonts from '../../styles/fonts';
+import resetCSS from '../../styles/reset';
+
+const initialStyles = `
+    ${resetCSS}
+    ${fonts.text}
+    background-color: #112F5C;
+`;
+
+const children = `
+    border: none;
+    background: transparent;
+    padding: 5px;
+    margin: -1px;
+    font-size: inherit;
+    color: inherit;
+    cursor: inherit;
+    display: flex;
+`
+
+const container = `
+    position: absolute;
+    width: 200px;
+    margin-left: -100px;
+    display: flex;
+    justify-content: center;
+    border-radius: 5px;
+    align-items: center;
+    left: 50%;
+    bottom: calc(100% + 5px);
+    pointer-events: none;
+`
+
+const box = `
+    position: relative;
+    color: #112F5C;
+    text-align: center;
+    border-radius: 5px;
+    padding: 10px 8px;
+    font-size: 14px;
```

Comment:
if this component holds text please use fonts.text and if you need a modifier like fonts.smallPrint please add that as the text will need the font family, line-height, etc etc. TY